src/Behat/MinkExtension/Context/MinkContext.php

```diff
@@ -238,22 +239,34 @@ public function assertResponseStatusIsNot($code)
 
     /**
      * Checks, that page contains specified text.
+     * @param string  $text    the text to check
+     * @param integer $timeout in milliseconds
      *
      * @Then /^(?:|I )should see "(?P<text>(?:[^"]|\\")*)"$/
      */
-    public function assertPageContainsText($text)
+    public function assertPageContainsText($text, $timeout = 10000)
```

Comment:
I'm not sure if adding timeout at this level at abstraction is useful. If we extend the idea, then timeout should be added to all functions on WebAssert class level, which would only complicate things.